docs/validator/validator-faq.md

```diff
@@ -14,32 +14,55 @@ Fisherman (or validator) - nodes that provide security by verifying the validity
-After this, a node that has a private key associated with the public key in the staking transaction must wait until epoch change to become validator if the staking amount is above the seat price.
+We call staking a process of sending StakeTransaction that inform the network that given account wants to become a validator in upcoming epochs. This particular type of transaction must provide a public key and staking amount.
+After the transaction is sent, a node that has a private key associated with the public key in the staking transaction must wait until the next epoch to become validator. **important**: a node can become a validator only if the amount in the staking transaction is above the seat price defined by the protocol.
+
+## What is a minimum amount to stake as a validator?
+
+Near, Inc. has staked 20 NEAR, so staking 10 NEAR should be enough to become a validator on the TestNet. On the MainNet, the minimum amount is dynamic, and is defined by the amount of NEAR tokens put at stake by other validators.
+
+## What is 'slashing'?
+
+In order to secure its Proof-of-Stake network, NEAR protocol punish the validators that sign two blocks with the same height (this is also defined as 'equivocation'). When this happens, the validator's funds at stake are destroyed, or 'slashed'.
+
+## Is NEAR enforcing liveness fault slashing?
+
+No. However, the protocol measures the uptime of each validator, and if the generated blocks are less than 90% of expected, the node will be kicked-out and lose its seat. In this case, a validator can re-stake after two epochs, and begin to sign blocks again after three epochs.
+
+## What is an Epoch?
+
+Epochs represent the interval of time (on the MainNet 12 hours) used by the protocol to:
+- Measure the performance and uptime of validators
+- Collect the bids from new validators
 
 ## What are the responsibilities of a validator?
 
-High level, validator must run node and be mostly online. If node is offline more than 10%, they will be removed from validator pool.
-As described above, in different roles of validators - the node running will be automatically selected to be one of the roles and perform it.
+High level, validators must run node and be mostly online. However, it's very important to keep private keys safe, otherwise adversaries might use them to sign malicious blocks, and trigger the protocol slashing.
+
+## Can I stake on a different shard?
+
+There's no way for a validator to decide the shard. The protocol randomically assigns validators to shards at the beginning of each epoch, and the node has one epoch to download its state. NEAR nodes have an automatic 'garbage collection' routine that deletes the state of previous shards after three epochs, to free up unused storage.
+Large validators will have to generate blocks signing across multiple shards, therefore it's important to size server and networking accordingly.
 
 ## How do I run a node?
 
 Follow [this tutorial.](local-setup/running-testnet.md)
```

Comment:
this link isn't as useful as the following 2 links

- [Becoming a Validator](/docs/validator/staking)
- [Running a Node](/docs/local-setup/running-testnet)